nautobot/utilities/testing/api.py

```diff
@@ -207,6 +207,28 @@ def test_options_objects(self):
             response = self.client.options(self._get_list_url(), **self.header)
             self.assertHttpStatus(response, status.HTTP_200_OK)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
+        def test_options_objects_returns_display_and_value(self):
+            """
+            Make an OPTIONS request for a list endpoint and validate choices use the display and value keys.
+            """
+            # Save self.user as superuser to be able to view available choices on list views.
```

Comment:
Is this really a restriction we have on the OPTIONS requests - superuser-only? That seems like something that should be changed.

Reply:
I'm not entirely sure it requires super user so you do bring up a good point, but potentially requires the `view_model_name` or whatever the default permissions are structured to use.

I can potentially define `self.user_permissions` to set the necessary `view` permissions on each test, but setting as superuser, I believe, is the easy button for these tests.

This is my uneducated guess.